tests/unit/components/Filter/FilterInput.spec.js

```diff
@@ -222,6 +222,22 @@ describe('FilterInput', () => {
     expect(wrapper.emitted('input')).toEqual([['/']]);
   });
 
+  it('selects input on focus if `selectInputOnFocus` prop is true', async () => {
+    wrapper = shallowMount(FilterInput, {
+      propsData: {
+        ...propsData,
+        selectInputOnFocus: true,
+        value: inputValue,
+      },
+    });
+    input = wrapper.find('input');
+    jest.spyOn(wrapper.vm, 'selectInputAndTags').mockImplementation();
+    input.setValue(inputValue);
```

Comment:
Minor comment: if you already set the input value when creating the wrapper, you shouldn't need to set it up here again.

Reply:
Thanks @marinaaisa 😄 I removed that line

Pr should be ready to go now